doozer/doozerlib/cli/scan_sources_konflux.py

```diff
@@ -104,6 +108,10 @@ async def run(self):
         for level in sorted(self.image_tree.keys()):
             await self.scan_images(self.image_tree[level])
 
+        # Check RHCOS status if the kubeconfig is provided
+        if self.ci_kubeconfig:
+            await self.detect_rhcos_status()
```

Comment:
Should ci_kubeconfig be required for ocp4_scan?

Reply:
Done with b7a13afc8bd30293772a29d4189fad11a25e12d2